sources/packages/forms/src/form-definitions/educationprogram.json

```diff
@@ -1,38 +1,35 @@
 {
-    "title": "Education Program",
+    "title": "Institution Welcome Page",
     "display": "form",
     "type": "form",
-    "name": "educationProgram",
-    "path": "educationprogram",
+    "name": "institutionwelcomepage",
```

Comment:
An form with name `institutionwelcomepage` in a file named `educationprogram` does not seems correct. Can you please check?

Reply:
ya thanks for the catch